hail/python/test/hailtop/batch/test_batch.py

```diff
@@ -406,9 +406,11 @@ def setUp(self):
         self.remote_tmpdir = remote_tmpdir
 
         if remote_tmpdir.startswith('gs://'):
-            self.bucket_name = re.fullmatch('gs://(?P<bucket_name>[^/]+).*', remote_tmpdir).groupdict()['bucket_name']
+            self.cloud_location = re.fullmatch('gs://(?P<bucket_name>[^/]+).*', remote_tmpdir).groupdict()['bucket_name']
         else:
-            self.bucket_name = None
+            assert remote_tmpdir.startswith('hail-az://')
+            storage_account, container_name = re.fullmatch('hail-az://(?P<storage_account>[^/]+)/(?P<container_name>[^/]+).*', remote_tmpdir).groups()
+            self.cloud_location = f'{storage_account}/{container_name}'
```

Comment:
sorry this is a nit, but can we we switch the words back to bucket for the tests too? Otherwise I'm good with this PR.